src/parser/core/modules/PrecastStatus.js

```diff
@@ -14,64 +14,87 @@ export default class PrecastStatus extends Module {
 
 	_combatantStatuses = {}
 	_combatantActions = []
+	_statusesToSynth = []
+	_actionsToSynth = []
+	_startTime = this.parser.fight.start_time
 
 	normalise(events) {
-		const startTime = this.parser.fight.start_time
-
 		for (let i = 0; i < events.length; i++) {
 			const event = events[i]
 			const targetId = event.targetID
 
+			const statusInfo = getDataBy(STATUSES, 'id', event.ability.guid)
```

Comment:
this could technically return `undefined` - probably safe to just `if (!statusInfo) { continue }` to prevent issues.

(checking `statusInfo.hasOwnProperty` if it's ever undefined will throw)